config.yaml

```
@@ -205,3 +205,17 @@ runtime:
     ## To just read the current rate, pass rate < 0.
     ## (For n>1 the details of sampling may change.)
     mutexProfileFraction: 0
+
+## synchronization should be enabled when multiple replicas of agent is deployed
+## if notification synchronization is enabled, then the active notification event-stream API
+## will get the notifications from multiple replicas
+synchronization:
+    notification:
+        enable: false
+        default: "redis"
+        pubsub:
+            redis:
+                host: "redis.demo.svc:6379"
+                password: ""
+                database: 0
+                channel: "optimizely-notifications"
```

What about this, so that we can use `pubsub` for any synchronization, not only for notification?

```suggestion
synchronization:
    pubsub:
        redis:
            host: "redis.demo.svc:6379"
            password: ""
            database: 0
            channel: "optimizely-notifications"
    notification:
        enable: false
        default: "redis"
    datafile:
        enable: false
        default: "redis"    
```

---

looks good, we might need a different channel for each service i.e. notification, datafile etc.